.travis.yml

```diff
@@ -22,6 +38,16 @@ jobs:
         - npm run lint-ts
     - name: "unit tests"
       script: npm test
+    - script:
+      name: "applitools tests for storybook with cypress 1t thread"
+      env:
+        - GROUP_NAME=chrome-visual
+      <<: *cypressVisual
+    - script:
+      name: "applitools tests for storybook with cypress 2d thread"
+      env:
+        - GROUP_NAME=chrome-visual
```

Comment:
The `- GROUP_NAME=chrome-visual` is always the same. We should just pass it in directly in the `package.json` (where we define `--chrome`).

Reply:
done